.github/workflows/CI.yml

```diff
@@ -132,6 +129,7 @@ jobs:
         env:
           JULIA_PKG_SERVER_REGISTRY_PREFERENCE: eager
       - uses: julia-actions/julia-runtest@v1
+        continue-on-error: ${{ matrix.version == 'nightly' || matrix.version == '1.10-nightly' }}
```

Comment:
we should consider 1.10 errors as errors

Reply:
Are 1.10-nightly builds stable? Or is it similar to nightly?

Reply:
it should be stable yeah